controllers/cloud.redhat.com/providers/kafka/strimzi.go

```diff
@@ -72,43 +269,48 @@ func NewStrimzi(p *p.Provider) (providers.ClowderProvider, error) {
 		},
 	}
 
+	// NOTE: we will not create ownerReferences on generated strimzi objects, since it is possible
+	// for them to reside a namespace that does not match the ClowdEnvironment's 'targetNamespace'
```

Comment:
I didn't think this would be a reason to not set the owner ref

Reply:
Set the owner ref as the ClowdEnvironment in https://github.com/RedHatInsights/clowder/pull/214/commits/ac9699b8ab0bc1459e395810731bf4e44d80a17f